sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileSourceStrategySuite.scala

```diff
@@ -528,6 +528,18 @@ class FileSourceStrategySuite extends QueryTest with SharedSparkSession with Pre
     }
   }
 
+  test("Add spark.sql.files.minPartitionNum config") {
```

Comment:
Shall we add `SPARK-32019: ` prefix into this test case name?

Reply:
Since it's not a bug.  So it's not need ?

Reply:
it's better to add it since it's a dedicated test case for this JIRA ticket.

Reply:
Add it.